platform/ui/src/components/Modal/Modal.jsx

```diff
@@ -41,20 +54,32 @@ const Modal = ({
 
   return (
     <ReactModal
-      className={classNames(className)}
+      className={classNames(
+        `relative py-6 w-11/12 lg:w-10/12 xl:w-1/2 max-h-full outline-none bg-primary-dark border border-secondary-main text-white rounded ${className}`
+      )}
       shouldCloseOnEsc={shouldCloseOnEsc}
+      onRequestClose={handleClose}
       isOpen={isOpen}
       title={title}
       style={customStyle}
     >
       <>
-        {renderHeader()}
-        <section>{children}</section>
+        <div className="px-6">{renderHeader()}</div>
+        <section
+          className="ohif-scrollbar overflow-y-auto px-6"
+          style={{ maxHeight: 'calc(100vh - 200px)' }}
```

Comment:
I created a custom CSS class `.modal-content` for it, and then I've used tailwind to get an existing theme value to avoid creating specific values in the tailwind config file:

https://github.com/OHIF/Viewers/blob/d5e20ddb98109c364b0961a74880bcd785909a16/platform/ui/src/components/Modal/Modal.css#L1-L3